infra/terraform/aws/with-new-vpc/main.tf

```diff
@@ -0,0 +1,57 @@
+terraform {
+  required_version = ">= 0.12.0"
+}
+
+provider "random" {
+  version = "~> 2.1"
+}
+
+provider "local" {
+  version = "~> 1.2"
+}
+
+locals {
+  cluster_name = "${var.name_prefix}-${random_string.suffix.result}"
+}
+
+resource "random_string" "suffix" {
+  length  = 8
+  special = false
+}
+
+locals {
+  public_subnet_tags ={
+    "kubernetes.io/cluster/${local.cluster_name}" = "shared"
+    "kubernetes.io/role/elb"                      = "1"
+  }
+  private_subnet_tags = {
+    "kubernetes.io/cluster/${local.cluster_name}" = "shared"
+    "kubernetes.io/role/internal-elb"             = "1"
+  }
+}
+
+module "vpc" {
+  source = "../modules/vpc"
+  region = var.region
+  name_prefix = var.name_prefix
+  private_subnet_tags = merge(var.private_subnet_tags, local.private_subnet_tags)
+  public_subnet_tags = merge(var.public_subnet_tags, local.public_subnet_tags)
+  tags = var.tags
+}
```

Comment:
I wonder if it would be simpler to just call the `terraform-aws-modules/vpc/aws` module here instead of wrapping it into our own.

Reply:
Yes, that makes sense. Will update it.